tests/components/swiss_public_transport/test_config_flow.py

```diff
@@ -52,22 +61,23 @@ async def test_flow_user_init_data_success(hass: HomeAssistant) -> None:
             user_input=MOCK_DATA_STEP,
         )
 
-        assert result["type"] is FlowResultType.CREATE_ENTRY
-        assert result["result"].title == "test_start test_destination"
+        assert result["type"] == FlowResultType.CREATE_ENTRY
+        assert result["result"].title == "test_start test_destination via via_station"
```

Comment:
Should also have some test with more than one `via` and probably without too.

Reply:
Ok, will add one without, with one and with many👍